Framework/HistogramData/src/Rebin.cpp

```diff
@@ -40,6 +40,13 @@ Histogram rebinCounts(const Histogram &input, const BinEdges &binEdges) {
     auto owidth = xo_high - xo_low;
     auto nwidth = xn_high - xn_low;
 
+    if (xo_high < -1e100 || xo_low < -1e100) {
```

Comment:
Neither would equal DBL_MIN - they come out as -DBL_MAX if the error I described occurs. The problem originates in ConvertUnits::getDetectorValues where, for want of a better value, efixed is set to DBL_MIN. The error then propagates through to here. I've made the check more explicit.